### PR TITLE
Apply 'user-mention-me' styles to self mentions span.

### PR DESCRIPTION
**Pull request by vishwesh3**
screen shot 2017-07-27 at 11 34 26 am

### CODE REVIEW

File: src/html/tags/HtmlTagSpan.js [hunk 1]
```diff
@@ -1,24 +1,45 @@
...
+import styles from '../HtmlStyles';
 
-export default class HtmlTagSpan extends PureComponent {
+class HtmlTagSpan extends PureComponent {
   props: {
     style?: StyleObj,
     actions?: Actions,
     cascadingStyle?: StyleObj,
+    attribs?: Object,
+    ownEmail: string,
```
Comment (borisyankov) [inline src/html/tags/HtmlTagSpan.js]: This is conceptually incorrect.
Our HtmlTagSpan should be aware only of general 'span' related stuff.
If we can not implement the styling by only generic code, we need to create a 'SpanMention' or something similar.

File: src/html/tags/HtmlTagSpan.js [hunk 2]
```diff
@@ -1,24 +1,45 @@
...
+  };
+
+  mentionButtonRefersToSelf = () => {
+    const { attribs, ownEmail } = this.props;
+    return attribs && attribs.class === 'user-mention' && attribs['data-user-email'] === ownEmail;
```
Comment (borisyankov) [inline src/html/tags/HtmlTagSpan.js]: Considering that the mention-self span already have a class called 'user-mention-me', this is likely too much complication.

File: src/html/tags/HtmlTagSpan.js (hunk 1 above)
Comment (vishwesh3) [inline src/html/tags/HtmlTagSpan.js]: Created new Component `HtmlTagSpanMention` .

**Comment by vishwesh3** [comment]
Updated

**Comment by borisyankov** [comment]
Looks good to merge.

**Pull request merged**